.github/build/Containerfile

```diff
@@ -5,10 +5,16 @@ LABEL summary="Toolchain for running pre-commit hooks." \
     io.k8s.display-name="Pre-Commit Toolchain"
 
 USER root
-RUN dnf install nodejs
+RUN dnf install nodejs -y && \
+    dnf clean all && \
+    rm -rf /var/cache/dnf
+ADD https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz $TMPDIR/
```

Comment:
I think that if we use ADD here, the Docker will create a separate image layer with tar file. And even if we delete it in the next command it will stay in the image. It will be only hidden by next layer. I would suggest to download tar with oc as part of the RUN command on next line.